README.md

```diff
@@ -212,6 +213,18 @@ rm -rf ani-cli
 
 For players you can use the apk (playstore/fdroid) versions of mpv and vlc. Note that these cannot be checked from termux so a warning is generated when checking dependencies.
 
+### iOS
+Install VLC from app store
```

Comment:
What's the reason for vlc by default again?

Reply:
> What's the reason for vlc by default again?

Mpv isnt available on iOS